server/src/crm/crm.service.ts

```diff
@@ -49,7 +70,7 @@ export class CrmService {
     return "Error";
   }
 
-  private dateReviver (key, value) {
```

Comment:
Not a blocker but I'd suggest we keep these `private` Typescript keywords for safety, especially since this is code that we didn't write ourselves.